recipe/meta.yaml

```diff
@@ -1,44 +1,64 @@
-{% set version = "2018_20170919" %}
+{% set vmajor = load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_VERSION_MAJOR +(\d+)").group(1)|int %}
+{% set vminor = load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_VERSION_MINOR +(\d+)").group(1)|int %}
+{% set vinterface = load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_INTERFACE_VERSION +(\d+)").group(1)|int %}
+{% set version = "%d.%d.%d"%(vmajor,vminor,vinterface%1000) %}
 
 package:
   name: tbb
   version: {{ version }}
 
 source:
-  fn: tbb{{ version }}oss_src.tgz
-  url: https://github.com/01org/tbb/archive/2018_U1.tar.gz
-  sha256: c6462217d4ecef2b44fce63cfdf31f9db4f6ff493869899d497a5aef68b05fc5
+  git_url: https://github.com/01org/tbb.git
+  git_tag: 2018_U2
 
 build:
-  number: 0
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  {% set args=' --build-args=stdver=c++11 --install-libs --install-devel --install-python --install-docs' %}
+  script: python build/build.py {{ args }}                                          # [unix]
+  script: python build/build.py --make-tool=mingw32-make {{ args }}                 # [win]
```

Comment:
we cannot build with mingw as it's a c++ lib ; we want msvc abi's

Reply:
we don't build with mingw, just using make program

Reply:
ok